### PR TITLE
removing ChennyBaBy from mailmap (reverting #3603)

### PR DESCRIPTION
**Pull request by 1ucian0**
ChennyBaBy requested to be removed from mailmap in https://github.com/Qiskit/qiskit-terra/pull/6420. Create a different PR so there is no need for CLA signing. 

Update: I reverting (most of) #3603,  where this entry was introduced, after reading the comments in it.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -16,87 +16,42 @@
...
 Antonio Mezzacapo <30698465+antoniomezzacapo@users.noreply.github.com> <amezzac@us.ibm.com>
-AzizNgoueya <49644465+AzizNgoueya@users.noreply.github.com>
-Borja Godoy Gago <Borja.Godoy.Gago@gmail.com>
-Bruno Schmitt <bruno.schmitt@epfl.ch>
-Cameron McGarry <cameron@cmcgarry.co.uk>
 Carmen Recio Valcarce <car_rv@outlook.es> <49230820+carmenrv@users.noreply.github.com>
-Charles Taylor <charlesastaylor@gmail.com>
 Chris Culver <chris.cb.culver@gmail.com>
 Christian Claus <cclauss@bluewin.ch>
-Christian Clauss <cclauss@me.com>
```
Comment (mtreinish) [inline .mailmap:42]: I think these 2 lines might be the same person and probably needs to be kept but updated to correctly map the commit authors as the same person. Probably something like:

```
Christian Clauss <cclauss@me.com> <cclauss@bluewin.ch>
```

@cclauss can you confirm?

**Review by mtreinish — COMMENTED** [review]
Overall LGTM, I found one entry in there being removed which probably needs to be modified instead of removed. But other than that this a good cleanup to the mailmap file. FWIW the author from #6420 removing the mailmap entry doesn't really change anything because that was just a duplicate of the commit metadata in the git log (which is immutable at this point).

As an aside we probably need to do another pass at fixing things looking through the log to check these removals there are quite a few author names that could use a cleanup or dedup.

File: .mailmap (hunk 1 above)
Comment (cclauss) [reply .mailmap:42]: Bluewin is dead...  Long live me.com.

Comment (1ucian0) [reply .mailmap]: done!

**Review by mtreinish — APPROVED** [review]
LGTM, thanks for the quick update

File: .mailmap [hunk 2]
```diff
@@ -28,7 +28,7 @@ Antonio D. Córcoles-Gonzales <adcorcol@us.ibm.com>
 Antonio Mezzacapo <30698465+antoniomezzacapo@users.noreply.github.com> <amezzac@us.ibm.com>
 Carmen Recio Valcarce <car_rv@outlook.es> <49230820+carmenrv@users.noreply.github.com>
 Chris Culver <chris.cb.culver@gmail.com>
-Christian Claus <cclauss@bluewin.ch>
+Christian Clauss <cclauss@me.com> <cclauss@bluewin.ch>
```
Comment (cclauss) [inline .mailmap:31]:
```suggestion
Christian Clauss <cclauss@me.com>
```

Comment (cclauss) [reply .mailmap:31]: Please

Comment (mtreinish) [reply .mailmap:31]: This is actually necessary, this is to map the old entries in the git log with `cclauss@bluewin.ch` to use your current email address and standardize your real name to be the same between the old commits and the current one.

**Pull request merged**